### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by karimelghazouly**
I have updated the install instruction to have the docker option

Description
I have updated the install instruction to have the docker option as we will be moving towards docker more in the future

Supporting information
BA
Dependencies
N/A

Screenshots
N/A

Testing instructions
NA

Deadline
N/A

### CODE REVIEW

File: documentation/installation.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ system. Also, remember to use **Python 3.5.x** or **Python 3.6.x**(use `pyenv`)
 vagrant environment as the playbooks only support them for now, and 
 **do not work** on Python 3.7 or higher.
 
+Also you can docker for development and the steps are mentioned [here|https://ocim.opencraft.com/en/latest/development/docker/]
```
Comment (saksham115) [inline documentation/installation.md:12]: you can use the*

**Review by saksham115 — CHANGES_REQUESTED** [review]
@karimelghazouly please update the PR description correctly

**Review by saksham115 — APPROVED** [review]
👍🏻 

- [ ]  I tested this: Nothing to test. Documentation changes.
- [ ]  I read through the code

**Comment by 0x29a** [comment]
This PR is no longer relevant. See the changes for https://github.com/open-craft/opencraft/pull/892.

**Pull request closed without merging**